### PR TITLE
[benchmarks] get skip tests based on device in runner.py and add some skip models

### PR DESCRIPTION
**Pull request by yudongsi**
skip some models which not suitable for benchmarks. @chuanqi129 

cc @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @ipiszy @msaroufim

### CODE REVIEW

File: benchmarks/dynamo/torchbench.py [hunk 1]
```diff
@@ -99,6 +100,9 @@ def setup_torchbench_cwd():
     "pyhpc_turbulent_kinetic_energy",
     "maml",
     "llama",
+    "doctr_det_predictor",
+    "doctr_reco_predictor",
+    "cm3leon_generate",
```
Comment (jgong5) [inline benchmarks/dynamo/torchbench.py:105]: This applies to both cpu and cuda? Consider to add reasons?

**Review by jgong5 — CHANGES_REQUESTED** [review]

File: benchmarks/dynamo/torchbench.py (hunk 1 above)
Comment (chuanqi129) [reply benchmarks/dynamo/torchbench.py:105]: Same reason with original `SKIP_TRAIN` models, there are no training implementation for those torchbench models for now

Comment (ESI-SYD) [reply benchmarks/dynamo/torchbench.py:105]: Yes, these models not designed for training now

**Review by desertfire — CHANGES_REQUESTED** [review]
We already do this at https://github.com/pytorch/pytorch/blob/c00dd43e43e9bdd041e44e5293c3fb8cd9e1bbf0/benchmarks/dynamo/common.py#L2942

**Comment by ESI-SYD** [comment]
script `run_{}_{}_{}_{}_{}.sh` generated by `generate_commands(suite)` seems not include models which from skip_for_cpu/cuda. Does it expected? https://github.com/pytorch/pytorch/blob/6bfd507c15f2d26212d3e2b9e581d9525bfd37d1/benchmarks/dynamo/runner.py#L423-L424

**Pull request closed without merging**